lms/djangoapps/courseware/static_tab.html

```diff
@@ -1,33 +0,0 @@
-## mako
```

Comment:
I discovered this stray file by accident. It is a template in the wrong directory so can never be used. I think it was an accident in the web fragments branch that this branch was rebased from, but when it was deleted the rebase reinstated it. I've deleted it in the parent branch so I'll rebase here and hope it goes away from the diff.